public/app/core/components/RolePicker/api.ts

```diff
@@ -1,15 +1,18 @@
 import { getBackendSrv, isFetchError } from '@grafana/runtime';
 import { Role } from 'app/types';
 
+import { addDisplayNameForFixedRole } from './utils';
+
 export const fetchRoleOptions = async (orgId?: number, query?: string): Promise<Role[]> => {
   let rolesUrl = '/api/access-control/roles?delegatable=true';
   if (orgId) {
     rolesUrl += `&targetOrgId=${orgId}`;
   }
-  const roles = await getBackendSrv().get(rolesUrl);
+  let roles = await getBackendSrv().get(rolesUrl);
   if (!roles || !roles.length) {
     return [];
   }
+  roles = roles.map(addDisplayNameForFixedRole);
```

Comment:
```suggestion
  return roles.map(addDisplayNameForFixedRole);
```
And then you can change `let` to `const` in declaration

Reply:
[48907cd](https://github.com/grafana/grafana/pull/62845/commits/48907cd9bab6eca9768ecc61f008330184d18d96)